src/ngx_http_lua_worker.c

```diff
@@ -99,6 +99,17 @@ ngx_http_lua_ffi_worker_pid(void)
 }
 
 
+int
+ngx_http_lua_ffi_master_pid(void)
+{
+#if (nginx_version >= 1013008)
+    return (int) ngx_parent;
+#else
+    return (int) ((ngx_process == NGX_PROCESS_SINGLE) ? ngx_pid : getppid());
```

Comment:
I think we should return an error here instead of returning a potential wrong value here.